server-external.js

```diff
@@ -30,6 +30,9 @@ const server = Hapi.server({
   cache: createCache(config)
 });
 
+// Add service connectors to request
+server.decorate('request', 'services', connectors);
```

Comment:
This code is making the external app API connectors available on the hapi 'request' objects.

The reason for this is that then although the pre-handlers are shared, it will be using the internal/external specific API connector without any need to pass configuration info.

It shouldn't be registering any routes.